erc20-load-test-tool/script/run-dapp.py

```diff
@@ -216,10 +216,14 @@ def aggregate_report(instance):
 # function to start wavefront proxy
 def start_wavefront_proxy():
     wavefront_token = os.environ['WAVEFRONT_TOKEN']
-    cmd = 'docker run -d -p 2878:2878 -e WAVEFRONT_URL=https://vmware.wavefront.com/api -e ' \
-          'WAVEFRONT_TOKEN=' + wavefront_token + ' -e JAVA_HEAP_USAGE=4G -e JVM_USE_CONTAINER_OPTS=false --name ' \
-                                                 'wavefront-proxy athena-docker-local.artifactory.eng.vmware.com' \
-                                                 '/wavefront-proxy:10.12 '
+    wavefront_username = os.environ['WAVEFRONT_USERNAME']
+    wavefront_pass = os.environ['WAVEFRONT_PASSWORD']
+
+    cmd = 'docker login vmwaresaas.jfrog.io -u ' + wavefront_username + ' -p ' + wavefront_pass + \
```

Comment:
How com wavefront credentials are used to login to `vmwaresaas.jfrog.io` ?

Reply:
Yes it is jfrog username and password, sure we can prerequisite docker login